ckanext/subakdc/templates/package/snippets/info.html

```diff
@@ -0,0 +1,18 @@
+{% ckan_extends %}
+
+{% block nums %}
+    {% if 'freshness_score' in h %}
+    {% set freshness_score = h.freshness_score(pkg) %}
+        {% if freshness_score > 0 %}
+            <div class="nums">
+                {# TODO replace inline style with w-full tailwind class #}
+                <dl style="width: 100%">
+                    <dt>{{ _('Dataset freshness') }} <i class="fa-question-circle" title="The dataset freshness score is a measure of how recently a dataset resource was updated."></i></dt>
```

Comment:
This could be more explicit, linking to the methodology in the plugin: 

'A score of 5 is given where last updated resource is < 180 days old
 A score falls in increments of 180 days (i.e a score of 1 for > 720 days or ~ 2 years)'

Reply:
Is the template the right place to include this doc though?